src/go/k8s/pkg/resources/certmanager/type_helpers.go

```diff
@@ -270,11 +271,28 @@ func (cc *ClusterCertificates) prepareAPI(
 
 	// for now we disallow having different issuer for each listener so that
 	// every time both listeners share the same set of certificates
-	nodeSecretRef := tlsListeners[0].GetTLS().NodeSecretRef
+	mainTLSListener := tlsListeners[0].GetTLS()
+
+	if mainTLSListener.ClientCACertRef != nil {
+		certName := NewCertName(cc.pandaCluster.Name, nodeCertSuffix)
```

Comment:
I think you can't do this, because there's no cert with this name and there won't be, right? instead we're using the secret ref provided.

I think I would rather deal with this in a way that I create another field like this one
```
// CR allows to specify node certificate, if not provided this will be nil
	externalNodeCertificate *corev1.ObjectReference
```

then if you follow up the usage of that, you will also have to copy the secret to your namespace, otherwise you can't mount it see `copyNodeSecretToLocalNamespace`